### ISSUE TITLE
Min/max don't work for lists of strings, but sort does.

### ISSUE THREAD

**Issue by BernhardWerner** [CindyJS/CindyJS#922]
As the title says: For an array like `array = ["abc", "xy"];` the functions `min` and `max` return `nada`. But! The function `sort` works and sorts the entries lexicographically. In Cinderella it is the same.

Firstly, I am strongly in favour of making these function operate the same on lists of strings. Secondly, I think it would be nice if they did work for lists of strings.

**Comment by kortenkamp** [comment]
I agree. There might be other data types that could be sorted (or even are already) and should have min/max, too, then.

One thing that bothers me a bit: I am not sure whether string sorting is locale dependent (I mean, it is, but is it implemented like that?).

**Comment by BernhardWerner** [comment]
> I agree. There might be other data types that could be sorted (or even are already) and should have min/max, too, then.

Oh, yes, I had the same thought after sending the message. I would probably do similar to how `sort` is implemented and support all types that it also supports. I'm currently working on these functions anyway.


> One thing that bothers me a bit: I am not sure whether string sorting is locale dependent (I mean, it is, but is it implemented like that?).

Oh, interesting question! I will take a look.
